core/csvcollector.class.inc.php

```diff
@@ -332,9 +332,9 @@ protected function Configure($aCsvHeaderColumns)
 	protected function CheckColumns($aSynchroColumns, $aColumnsToIgnore, $sSource)
 	{
 		Utils::Log(LOG_DEBUG, "[".get_class($this)."] Columns [".var_export($aSynchroColumns, true)."]");
-		foreach ($this->aFields as $sSynchroColumn => $aDefs) {
+		foreach ($this->aFields as $sField => $aDefs) foreach ($aDefs['columns'] as $sSynchroColumn) {
 			if (array_key_exists($sSynchroColumn, $this->aSynchroFieldsToDefaultValues) || in_array($sSynchroColumn, $this->aIgnoredSynchroFields)) {
-				$aColumnsToIgnore[] = $sSynchroColumn;
+				$aColumnsToIgnore[] = $sField;
```

Comment:
I am not really sure of this change, as it is not really clear in the documentation what 'ignore' actually means.